src/Components/ShowAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function ShowAllArticles({ selectedArticles }) {
+  const [allArticles, setAllArticles] = useState([]);
+  const [IsLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(`https://nc-news-nikhil.herokuapp.com/api/articles`)
```

Comment:
I'd think about using Axios instead, and putting your api calls into a new folder